### PR TITLE
Update user guide for ExampleGen downstream components

### PR DESCRIPTION
**Pull request by tinally**
# Update user guide on custom split configuration for ExampleGen downstream components.

### CODE REVIEW

File: docs/guide/examplegen.md [hunk 1]
```diff
@@ -284,3 +284,117 @@ from tfx.examples.custom_components.presto_example_gen.presto_component.componen
...
+
+### StatisticsGen
+
+Default behavior is to perform stats generation for all splits.
+
+To exclude any splits, set the `exclude_splits` for StatisticsGen component.
+For example:
+
+```python
+from tfx import components
```
Comment (1025KB) [inline docs/guide/examplegen.md:300]: import the component directly

**Comment by tinally** [comment]
I read through the entire guide and updated the parts that don't align with head implementation. If you find anything else that should be updated, let me know. Thanks!

File: docs/guide/examplegen.md [hunk 2]
```diff
@@ -5,8 +5,10 @@ consumes external files/services to generate Examples which will be read by
 other TFX components. It also provides consistent and configurable partition,
 and shuffles the dataset for ML best practice.
 
-*   Consumes: Data from external data sources such as CSV, `TFRecord` and BigQuery
-*   Emits: `tf.Example` records
+*   Consumes: Data from external data sources such as CSV, `TFRecord`, Arvo,
```
Comment (rcrowe-google) [inline docs/guide/examplegen.md:8]: Avro, not Arvo

Comment (tinally) [reply docs/guide/examplegen.md]: thanks for the catch!

**Pull request merged**